animations/lib/src/basics/09_animated_switcher.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+double generateHeight() => Random().nextDouble() * 200;
+
+double generateWidth() => Random().nextDouble() * 200;
+
+Widget generateContainer() => Container(
```

Comment:
This doesn't really need to be a separate method, since it's only used in one place (`changeWidget`), and that method's just wrapping this one in `setState`. I would just inline it there. 

Also, consider adding more properties and making them random, such as color, border radius, etc. That will help make the differences between widgets more obvious during the animation.